tests/test_exceptions.py

```
@@ -0,0 +1,33 @@
+
+from unittest.mock import MagicMock
+from apps.exceptions import handle_view_exceptions
```

Ruff (0.8.2)

`apps.exceptions.handle_view_exceptions` imported but unused

Remove unused import: `apps.exceptions.handle_view_exceptions`

(F401)